themes/index.js

```diff
@@ -119,6 +119,12 @@ const themes = {
     text_color: "FFFFFF",
     bg_color: "050F2C",
   },
+  "gatsby":{
```

Comment:
maybe the name "gatsby" can also refer to gatsbyjs so i think it'll be better if we change it to something else. great-gatsby?

Reply:
Yes, that change would be great!